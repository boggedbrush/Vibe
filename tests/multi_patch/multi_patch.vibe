# VibeSpec: 1.5

# Replace the existing greet()
patch_type: add_function
file: hello.py
--- code: |
    def greet(name):
        print(f"Greetings, {name}!")

# Then add a new function
patch_type: add_function
file: hello.py
--- code: |
    def farewell(name):
        print(f"Goodbye, {name}!")
