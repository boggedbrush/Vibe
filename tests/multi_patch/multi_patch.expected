# hello.py
def greet(name):
    print(f"Greetings, {name}!")

def farewell(name):
    print(f"Goodbye, {name}!")

class Greeter:
    def greet(self):
        print("Greeter says hi!")
